README.md

```diff
@@ -381,54 +381,54 @@ Built-in functions. Ready to be bound to any key you like. :smile:
-| `builtin.current_buffer_tags`       | Lists Tags in current buffer.                                                               |
+| Functions                           | Description                                                                                                      |
+|-------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| `builtin.buffers`                   | Lists open buffers in current neovim instance                                                                    |
+| `builtin.oldfiles`                  | Lists previously open files                                                                                      |
+| `builtin.commands`                  | Lists available plugin/user commands and runs them on <cr>                                                       |
+| `builtin.tags`                      | Lists tags in current directory with preview (ctags -R)                                                          |
+| `builtin.command_history`           | Lists commands that were executed recently, and reruns them on <cr>                                              |
+| `builtin.search_history`            | Lists searches that were executed recently, and reruns them on <cr>                                              |
+| `builtin.help_tags`                 | Lists available help tags and opens a new window with the relevant help info on <cr>                             |
+| `builtin.man_pages`                 | Lists manpage entries, opens them in a help window on <cr>                                                       |
+| `builtin.marks`                     | Lists vim marks and their value                                                                                  |
+| `builtin.colorscheme`               | Lists available colorschemes and applies them on <cr>                                                            |
+| `builtin.quickfix`                  | Lists items in the quickfix list                                                                                 |
+| `builtin.loclist`                   | Lists items from the current window's location list                                                              |
+| `builtin.vim_options`               | Lists vim options, allows you to edit the current value on <cr>                                                  |
+| `builtin.registers`                 | Lists vim registers, pastes the contents of the register on <cr>                                                 |
+| `builtin.autocommands`              | Lists vim autocommands and goes to their declaration on <cr>                                                     |
+| `builtin.spell_suggest`             | Lists spelling suggestions for the current word under the cursor, replaces word with selected suggestion on <cr> |
```

Comment:
😯 This is new to me

Reply:
Honestly the best part of writing these docs, outside of providing better instructions for the community :smile:, is that I learned about several features and pickers I didn't even know existed! Big win in my book :sunglasses: